registry.k8s.io/images/k8s-staging-etcd/images.yaml

```diff
@@ -37,7 +37,540 @@
+    "sha256:12e52057f886440da1c12b12175d2765e0e9e1ce91996184aa98e00eaaa44f6a": ["v3.6.0-rc.4-amd64"]
+    "sha256:1047bad1e0ffe59eb05935306d6e9dc41e68077df0a1a8bd6f08edcf90cf9fbf": ["v3.6.0-rc.4-arm64"]
+    "sha256:663ec1d990decf9701ac113cbd20aa5ab70a60b1c0354bd8881e0e01f69f5f4c": ["v3.6.0-rc.4-ppc64le"]
+    "sha256:10b811b3847e2949fae28177a4d78dda9da8fd34df8601aa0ee213b3437f5cd4": ["v3.6.0-rc.4-s390x"]
+    "sha256:8c0bebcbe491032f9cc5886b4627343fd75ee8fef456b68826b5b8509571b7c9": ["v3.6.0-rc.5"]
+    "sha256:61d03cfa9cde0e1c083f143c72c67dd6b109486c4e440408db108e70006fdcff": ["v3.6.0-rc.5-amd64"]
+    "sha256:29afc072cf52a2d735e5e509ac308ad6aa03776161661af881d8b81d8689214b": ["v3.6.0-rc.5-arm64"]
+    "sha256:5219a0f05e828330a9452f89925ede6c3349e8f10116ca4e4d5d441442b28a36": ["v3.6.0-rc.5-ppc64le"]
+    "sha256:8779cf8437cd48fb7c0427da13a368229485a38ae7c438ff89cc9d1376d3888c": ["v3.6.0-rc.5-s390x"]
+    "sha256:3cdb912f9d7e4c087b3c815b2d04ba410d115b4afad8fc3123e3f438ae8df664": ["v3.6.0-s390x"]
+    "sha256:56bc6ef112a4e2702142d365baecae44cba85c6efc6146b4a41a4c655dd7d231": ["v3.6.1"]
+    "sha256:9a092b7ecb80e499a88e7967a18238234c9baa733233456f92dc9e7a18ef27df": ["v3.6.1-amd64"]
+    "sha256:3aed53b019171e409052d0e307f3425496eaa555eabb5fe82f9fa73c27302238": ["v3.6.1-arm64"]
+    "sha256:e73062c74d8d410f6ea8289641caa60a6ce0c632c260dc697ded3da7b379d797": ["v3.6.1-ppc64le"]
+    "sha256:c3e862c25403099045c8503a9afccf535fb9d16d52fc05bfd083239090203216": ["v3.6.1-s390x"]
+    "sha256:6adf168b4454b1f62377e2ec356455f64e8bcff6942e1a23e73129e46dc281d2": ["v3.6.2"]
+    "sha256:282333c2e70fe23fecb7e3fa487e622479649d61bf2c90bc26fba8fc44a6effb": ["v3.6.2-amd64"]
+    "sha256:6c412ccab1cbcb17a6ff344987497418fd7b72b51161790987922d03f6573a3a": ["v3.6.2-arm64"]
+    "sha256:6884179d112beef6d2b374ecb34a93974987d89cc2f2d9173498d66e3b9db004": ["v3.6.2-ppc64le"]
+    "sha256:902de3fefa0e4caec6d6afb820f2b4095c80dce56d1c5b0afa9f213f9fb3aa36": ["v3.6.2-s390x"]
+    "sha256:7ab0d37cafc5e654f5e268aaf86c623fe046af0e8a72c54b0052884246cea760": ["v3.6.3"]
+    "sha256:7cfba2f772895a27c659e6919d6bb87eb1b0d900659915da2e6629cfb5ddece3": ["v3.6.3-amd64"]
+    "sha256:b37026ad9a6447ef68c2f20ac9371b5c2630752093ead5e2c9dc9711b179c6f5": ["v3.6.3-arm64"]
+    "sha256:4a4a9f1ff86cd5076d519f4fcf72320262410f929ed2e520e5378c0777d764c8": ["v3.6.3-ppc64le"]
+    "sha256:f270dacfeeec93674dcf6300bf858f8f2b9b4e988b137b1c5d3741be177bd09b": ["v3.6.3-s390x"]
+    "sha256:5d10878e4fd4ebfdf82bc142fb044542a3ca514c0ee169277643a84d6816892a": ["v3.6.4"]
+    "sha256:ab22e11ccb7b32df5d25e4ac8ea5cded2a9597e32e04ad17799b43694d7a848e": ["v3.6.4-amd64"]
+    "sha256:2f89f31d35636579b1f73a0450d6454e10c664a4c57b8d7fc552077ea05e8227": ["v3.6.4-arm64"]
+    "sha256:6372191f9cb9b529daf7322b933b76c46275b5deca7d7fc99819b66b13c81bbc": ["v3.6.4-ppc64le"]
+    "sha256:0aa5dd4ea7e2fdd5bd69ba0d467f7a8e605ca1ec59c19bb50acd0e3655d60fdc": ["v3.6.4-s390x"]
```

Comment:
The v3.6.4 is multi-arch manifest, should we only add v3.6.4 or all v3.6.4-[arch] ?

Reply:
As long as they were published in the official registry, not sure if they can be removed without a release note and a new minor release...

Reply:
@ameukam they were already promoted in the official registry. If this will become the source of truth in the future, it may need all tags.

Reply:
actually, they can always be added later if needed...